preprocess.py

```diff
@@ -300,12 +235,6 @@ def consumer(tensor):
     return res
 
 
-def binarize_with_load(args, filename, dict_path, output_prefix, lang, offset, end):
```

Comment:
Thanks for removing these, they somehow crept in from other internal users, but should be moved to those call sites directly since they aren't used in fairseq.